packages/smooth_app/lib/pages/scan/scan_product_card.dart

```diff
@@ -15,7 +15,6 @@ class ScanProductCard extends StatelessWidget {
     final ProductPreferences productPreferences =
         context.watch<ProductPreferences>();
     return GestureDetector(
-      onTap: () => _openProductPage(context),
```

Comment:
I have the same opinion as @teolemon, I think it's not compfortable to have click on the small text on the bottom